### PR TITLE
[IT-1007] VPN access

### PR DESCRIPTION
**Pull request by zaro0508**
Setup jumpcloud group access to VPCs after connecting to the VPN.

### CODE REVIEW

File: config/prod/sage-client-vpn.yaml [hunk 1]
```diff
@@ -32,3 +32,36 @@ sceptre_user_data:
       AccessGroups:
         - "admin"
         - "scientists"
+    # org-sagebase-scicomp
```
Comment (ahayden) [inline config/prod/sage-client-vpn.yaml:35]: Is it possible to define the role to network mapping in a YAML map, and for the map to be referable to other templates? It seems like we are having to store cidr ranges, group mapping, and vpc names in too many places.

File: config/prod/sage-client-vpn.yaml [hunk 2]
```diff
@@ -32,3 +32,36 @@ sceptre_user_data:
...
+      CIDR: "10.29.0.0/16"
+      AccessGroups:
+        - "admin"
+        - "aws-scicomp-developers"
+    # org-sagebase-strides
+    stridespoolvpc:
+      CIDR: "10.29.0.0/16"
+      AccessGroups:
+        - "admin"
+        - "aws-scicomp-developers"
```
Comment (ahayden) [inline config/prod/sage-client-vpn.yaml:67]: This should probably have a distinct group just for people who need access to strides resources

**Review by ahayden — CHANGES_REQUESTED** [review]

File: config/prod/sage-client-vpn.yaml (hunk 1 above)
Comment (zaro0508) [reply config/prod/sage-client-vpn.yaml:35]: maybe?  but i think we can update with a follow on PR?

File: config/prod/sage-client-vpn.yaml (hunk 2 above)
Comment (zaro0508) [reply config/prod/sage-client-vpn.yaml:67]: I think you can make that update as a follow on PR?

**Review by ahayden — APPROVED** [review]
Sounds good

**Pull request merged**